libs/SalesforceSDKCore/SalesforceSDKCore/Classes/IDP/SFSDKLoginFlowSelectionViewController.m

```diff
@@ -233,20 +231,14 @@ - ( UIImage * _Nonnull )headerBackgroundImage {
 }
 
 - (IBAction)useIDPAction:(id)sender {
-    if ([self.selectionFlowDelegate respondsToSelector:@selector(loginFlowSelectionIDPSelected:)]) {
```

Comment:
Should `loginFlowSelectionIDPSelected` be optional in the protocol?
What happens if it is not there?